config/router.config.js

```diff
@@ -74,12 +74,27 @@ module.exports = [
             icon: 'dashboard',
             component: './Overview',
           },
+          {
+            path: '/private/overview',
+            name: 'overview',
+            component: './Overview',
+          },
           {
             path: '/private/profile',
             name: 'profile',
             exact: true,
             component: './Profile',
           },
+          {
+            path: '/private/result',
```

Comment:
We may want to consider isolating the context of user authentication to the internals of the dashboard. For example, naming the route `/overview/result` in this case.

Reply:
@gurbirkalsi `/private` is added here because of the `private` wrapper inside which the routes are present. We won't be able to access the route without add `/private`  as a prefix to it. Either we need to delete the wrapper and handle the private vs public routes in some other way or rename the wrapper itself.